pkg/docker/docker.go

```diff
@@ -318,7 +331,7 @@ func getDefaultContext(timeout time.Duration) (context.Context, context.CancelFu
 // GetImageWorkdir returns the WORKDIR property for the given image name.
 // When the WORKDIR is not set or empty, return "/" instead.
 func (d *stiDocker) GetImageWorkdir(name string) (string, error) {
-	resp, err := d.kubeDockerClient.InspectImage(name)
+	resp, err := d.InspectImage(name)
```

Comment:
Gotcha - for what it is worth, I think 2) is the better long term solution, and at first blush think it is less painful in the short term as well.